file_management.py

```diff
@@ -0,0 +1,184 @@
+from PIL import Image
+from abc import ABC, abstractmethod
+import json
+import os
+import boto3
+import io
+from pdf2image import convert_from_bytes, convert_from_path
+import shutil
+
+
+class FileManagement(ABC):
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def get_image_file_name(self, file_path):
+        pass
+
+    @abstractmethod
+    def move_files(self, source_key, destination_key):
+        pass
+
+    @abstractmethod
+    def get_label_folder(self):
+        pass
+
+    @abstractmethod
+    def list_files_in_folder(self, folder_path):
+        pass
+
+    @abstractmethod
+    def create_folder(self, folder_path):
+        pass
+
+    @abstractmethod
+    def read_file(self, file_path):
+        pass
+
+    @abstractmethod
+    def read_document(self, file_path):
+        pass
+
+    @abstractmethod
+    def save_current_state(self, file_name, bounding_boxes, label_folder_path):
+        pass
```

Comment:
You can use these functions to implement all the code that is similar in both cases and call this with super() in the daugther classes. Ask me if you don't understand.
I didn't look yet, but in the next commit I will looke at the functions and I don't want a single line of code duplication.